### ISSUE TITLE
No pagination is avaliable for GOV.UK Frontend review application page

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1414]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue30
**URL**: http://govuk-frontend-v3.herokuapp.com/full-page-examples/news-and-communications?order=most-viewed
**Page**: News and communications

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58716331-b4b17900-83c0-11e9-87c8-48e2b2abb9f9.png)

The page tested stated that there were 822,411 results, however as these cannot be displayed on one page, a pagination feature should be present to allow users to navigate through the other pages of results. Analysts felt that more results must be available but
were unable to view them. Even with the filter options selected, that many results would not be lessened by enough to display on one page.

## Screen reader comments
> “No feature to enable me to view the next page of results was present on the page. This would mean in a non-testing environment that I was able to view the first page of results only. Implementing a pagination to enable me to view different pages would provide a more effective experience. This occurred using JAWS, NVDA and VoiceOver for iPhone and Mac.”

**Comment by nickcolley** [comment]
This review page is intentionally fake, so pagination would not really function as expected, we could consider adding one to make the example more realistic but might be more effort than needed here.

I've but awaiting-triage on this to make a decision on this, the priority is low since it's not an issue with GOV.UK Frontend.

**Comment by nickcolley** [comment]
Met with the developers and we have decided to close this given the time it'd take to update, and this is intentionally fake.
